bindings/postgres/metadata.yaml

```diff
@@ -0,0 +1,29 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: v1
+type: bindings
+name: postgres
+version: v1
+status: stable
+title: "PostgreSQL"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-bindings/postgres/
+binding:
+  output: true
+  input: false
+  operations:
+    - name: exec
+      description: "The exec operation can be used for DDL operations (like table creation), as well as INSERT, UPDATE, DELETE operations which return only metadata (e.g. number of affected rows)."
+    - name: query
+      description: "The query operation is used for SELECT statements, which returns the metadata along with data in a form of an array of row values."
+    - name: close
+      description: "The close operation can be used to explicitly close the DB connection and return it to the pool. This operation doesn’t have any response."
+metadata:
+  - name: url
+    required: true
+    sensitive: true
+    description: "The PostgreSQL binding uses pgx connection pool internally so the url parameter can be any valid connection string, either in a DSN or URL format."
```

Comment:
I wouldn't get into the details of pgx here. What I'd say is:

`Connection string for PostgreSQL`

Then you can add:

```yaml
url:
  title: More details
  url: https://docs.dapr.io/reference/components-reference/supported-bindings/postgres/#url-format
```

Reply:
Fixed